README.md

```diff
@@ -1,3 +1,101 @@
-# MoveSingleAPISwift
+# MoveSingleAPI SDK in Swift
 
-A description of this package.
+## Installation 💻
+You can use Swift Package Manager to integrate the library by adding the following dependency in the `Package.swift` file or by adding it directly within Xcode.
+
+`.package(url: "https://github.com/move-ai/MoveSingleAPISwift", from: "1.0.0")`
+
+## Usage 🤩
+Import the framework in your project:
+```swift
+import MoveSingleAPISwift
+```
+
+[Create an MoveSingleAPI key](https://move.ai) and add it to your configuration:
+```swift
+let move = Move(apiKey: "TOKEN")`
+```
+
+### Create simple Take
+```swift
+let take = try await move.createTake(
+    takeID: UUID().uuidString, 
+    videoURL: // URL to local video file
+)
+```
+
+### Upload Take
+Uploads the video file and the meta data and creates a new take in the system.
+```swift
+try await take.upload()
+```
+
+### Create Job
+Take has to be uploaded. This creates a new job that runs the video through the pipeline to generate the animation.
```

Comment:
Can we add a line explaining what a job is, using the copy from the Python SDK :
"Jobs are the processing entity in MoveUGC. By creating a Job, you can initiate the processing for a take. A job can have multiple output types, currently the only output types supported are mp4, usdc and fbx files."